fastlane/docs/Actions.md

```diff
@@ -886,7 +886,7 @@ See how [Artsy](https://github.com/fastlane/examples/blob/master/Artsy/eidolon/F
 ### [Crashlytics Beta](http://try.crashlytics.com/beta/)
 ```ruby
 crashlytics(
-  crashlytics_path: './Crashlytics.framework', # path to your 'Crashlytics.framework'
+  crashlytics_path: './Pods/Crashlytics/', # path to your Crashlytics submit binary.
```

Comment:
I don't have a project available right now, but shouldn't it be `./Pods/iOS/Crashlytics/Crashlytics.framework`?